modules/modal/react/stories/stories.tsx

```diff
@@ -25,7 +25,56 @@ const DefaultModalExample = () => {
       <Button variant={Button.Variant.Delete} buttonRef={buttonRef} onClick={openModal}>
         Delete Item
       </Button>
-      <Modal data-testid="TestModal" heading="Delete Item" open={open} handleClose={closeModal}>
+      <p>
+        <a href="#">Link</a>
+      </p>
+
+      <button type="button">Button</button>
```

Comment:
Why do we need all this extra content?

Reply:
Accessibility requested this for testing if content was really being hidden from assistive technology

Reply:
should i add this back in?

Reply:
Maybe do that in an accessible story? Maybe behind a knob setting called `extraElements` that defaults to `false`? I don't think it is strictly necessary. We don't have a way to test this automatically, but it is nice for manual verification.